tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/api/PostResource.java

```diff
@@ -308,11 +309,10 @@ public ResponseEntity<List<PlacementSummaryDTO>> getPlacementsForPosts(
    * @return the ResponseEntity with status 200 (OK)
    */
   @DeleteMapping("/posts/{id}")
-  @PreAuthorize("hasAuthority('tcs:delete:entities')")
+  @PreAuthorize("hasAuthority('post:delete:posts')")
```

Comment:
I have tested locally with the profile service. With necessary changes in the profile DB and code it works fine. Refer to profile PR

Reply:
Changed